src/components/staff-payment/StaffPayment.vue

```diff
@@ -98,22 +94,20 @@
             label="No Fee"
             :value="StaffPaymentOptions.NO_FEE"
           />
-
-          <template v-if="displayPriorityCheckbox">
-            <v-divider class="mt-6" />
-
-            <!-- Priority checkbox -->
-            <v-checkbox
-              id="priority-checkbox"
-              class="priority-checkbox mt-6 pt-0"
-              label="Priority (add $100.00)"
-              hide-details
-              :input-value="staffPaymentData.isPriority"
-              :disabled="paymentOption === StaffPaymentOptions.NO_FEE"
-              @change="emitStaffPaymentData({ isPriority: !!$event })"
-            />
-          </template>
```

Comment:
This had to be moved outside of the `v-radio-group`. For some reason it was binding to `staffPaymentData.option` and changing it's value when clicked.